packages/node/src/serverless-functions/helpers-web.ts

```diff
@@ -1,26 +1,7 @@
 import type { ServerResponse, IncomingMessage } from 'http';
 import type { NodeHandler } from '@edge-runtime/node-utils';
 import { buildToNodeHandler } from '@edge-runtime/node-utils';
-
-class FetchEvent {
-  public request: Request;
-  public awaiting: Set<Promise<void>>;
-  public response: Response | null;
-
-  constructor(request: Request) {
-    this.request = request;
-    this.response = null;
-    this.awaiting = new Set();
-  }
-
-  respondWith(response: Response) {
-    this.response = response;
-  }
-
-  waitUntil() {
-    throw new Error('waitUntil is not implemented yet for Node.js');
-  }
-}
+import Edge from '@edge-runtime/primitives';
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (blocking):** ‏add tests

Can we have a couple tests that ensure the use of this updated `FetchEvent` is working correctly? I think at least having the following would be awesome.

- test a serverless function with web handler, like [the edge function test](https://github.com/vercel/vercel/blob/142a397d8eb9575d6ae17ec36b0d1a021a2dfc5a/packages/node/test/unit/dev.test.ts#L311)
- test the updated isStreaming logic properly detects those cases with web handlers